pkg/snapshot/snapshot_test.go

```diff
@@ -165,23 +168,6 @@ func TestSnapshotFiles(t *testing.T) {
 	testutil.CheckErrorAndDeepEqual(t, false, nil, expectedFiles, actualFiles)
 }
 
-func TestEmptySnapshotFS(t *testing.T) {
```

Comment:
y no longer a test case to test what happens when there is nothing to snapshot?

Reply:
The special casing was removed so now a tar is always created, it's just empty. I added the test back with that assertion.